test/masq.test.js

```diff
@@ -501,4 +500,121 @@ describe('masq protocol', function () {
 
     masq.handleUserAppLogin('channel', keyBase64, 'someAppId')
   })
+  // it('should delete the userApp-db after masqAccessGranted message because of a connection failure', (done) => {
+  //   const channelName = 'channel111'
+  //   const hub = signalhub(channelName, 'localhost:8080')
+  //   const sw = swarm(hub)
+  //   let dbName = null
+
+  //   sw.on('close', async () => {
+  //     await wait(20000)
+
+  //     expect(await dbExists(dbName)).to.be.false
+  //     setTimeout(done, 1000)
+  //   })
+
+  //   sw.once('peer', peer => {
+  //     peer.once('data', async (data) => {
+  //       const { msg } = await decrypt(cryptoKey, JSON.parse(data), 'base64')
+  //       expect(msg).to.equal('notAuthorized')
+  //       peer.once('data', async (data) => {
+  //         const { msg, userAppDbId } = await decrypt(cryptoKey, JSON.parse(data), 'base64')
+  //         expect(msg).to.equal('masqAccessGranted')
+
+  //         dbName = `app-${masq.profileId}-${userAppDbId}`
+  //         expect(await dbExists(dbName)).to.be.true
+
+  //         sw.close()
+  //       })
+
+  //       const message = {
+  //         msg: 'registerUserApp',
+  //         name: 'test app',
+  //         description: 'description goes here',
+  //         imageURL: ''
+  //       }
+  //       const encryptedMsg = await encrypt(cryptoKey, message, 'base64')
+  //       peer.send(JSON.stringify(encryptedMsg))
+  //     })
+  //   })
+
+  //   masq.handleUserAppLogin(channelName, keyBase64, `someAppId${channelName}`)
+  //     .then(() => masq.handleUserAppRegister(true))
+  // })
+  it('should clean after masqAccessGranted message and start the register again succesfully', (done) => {
```

Comment:
I don't think this test asserts the db is cleaned ?

Reply:
or should this test only asserts that two consecutives registers work ok ?

Reply:
yes this test normally assumes that the previous test is ok, i.e the db deletion

Reply:
So we should remove this test as the db deletion is not tested yet ?

Reply:
yes, removed.